spec/proceduralSpec.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace procedural\spec;
+
+use function prodecural\echoScript;
+
+describe('procedural', function () {
+  describe('->echoScript()', function () {
+    it('echos input string inside a script tag', function () {
```

Comment:
I'm not following. Could you please link to an LOC in the code matrix? I don't see how this is relevant to this pull any longer, and I'm starting to feel like the scope is creeping due to unexpected upstream additions during development.